lib/libs/email/content/withdrawRai/emailTemplates/AppKCMS.tsx

```diff
@@ -1,28 +1,30 @@
-import { CommonEmailVariables } from "shared-types";
-import { RaiWithdraw } from "shared-types";
-import { Html, Container } from "@react-email/components";
-import { WithdrawRAI, PackageDetails, BasicFooter } from "../../email-components";
+import { CommonEmailVariables, Events } from "shared-types";
+import { Attachments, PackageDetails, BasicFooter, SpamWarning } from "../../email-components";
+import { BaseEmailTemplate } from "../../email-templates";
 
 export const AppKCMSEmail = (props: {
-  variables: RaiWithdraw & CommonEmailVariables;
+  variables: Events["RespondToRai"] & CommonEmailVariables;
   relatedEvent: any;
```

Comment:
What's `relatedEvent`?

Reply:
`WithdrawRAI` the kafka event doesn't have the users name or email stored on the variable, so Ben is fetching the event prior `Respond to Rai` in order to retrieve the missing data we need. 
I added a type in the same folder as the `CommonEmailVariables` though so we aren't using the type `any` in those cases. Thanks for pointing it out!